db/seeds/seed.js

```diff
@@ -68,6 +71,20 @@ const seed = ({ itemData, userData, favouritesData }) => {
 
       return basketTable;
     })
+    .then(() => {
+      const recent_items = db.query(
+        `
+        CREATE TABLE recent_items (
+          recent_id SERIAL PRIMARY KEY,
+          clothes_id INT NOT NULL REFERENCES clothes(clothes_id),
+          uid VARCHAR NOT NULL REFERENCES users(uid),
+          created_at TIMESTAMP NOT NULL DEFAULT NOW()
+        );
+        `
+      );
+
```

Comment:
plus one Many-to-Many relationship between Clothes and Users, hope this is the last one :)